oada-core/http-handler/oada-srvc-http-handler/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "ES2018",
+    "strict": true,
+    "esModuleInterop": true,
```

Comment:
I recommend we turn this off if we can. Most things can be correctly imported without it now.

Reply:
It is needed for ajv

Reply:
AJV can be imported as `import * as Ajv from "ajv"` without it.